cpp/include/mcts/SearchLog.hpp

```diff
@@ -17,6 +17,7 @@ class SearchLog {
   using ValueArray = Game::Types::ValueArray;
   using node_index_t = int;
   using edge_index_t = int;
+  using player_bitset_t = std::bitset<Game::Constants::kNumPlayers>;
```

Comment:
Not good to define the same type twice in the code-base like this. Please move this to `GameTypes`, and then do `using player_bitset_t = Game::Types::player_bitset_t`. Grep for "using player_bitset_t" and replace other instances.